File: src/com/turlington/iNotify.java
package com.turlington;

/**
 * What to do when a job is found.
 * Created by Mitchell on 7/18/2016.
 */
public interface iNotify {

    /**
     * Announce that a job has been found.
     * @param jobListing
     */
    void announce(JobListing jobListing);
}

File: src/main/kotlin/com/turlington/iNotify.kt
package com.turlington

/**
 * What to do when a job is found.
 * Created by Valerie on 7/18/2016.
 */
interface iNotify {
    /**
     * Announce that a job has been found.
     */
    fun announce(jobListing: JobListing)
}
